pkg/sidecar/config.go

```diff
@@ -31,6 +31,7 @@ type Config struct {
 	SvidBundleFileNameDeprecated       string `hcl:"svidBundleFileName"`
 	RenewSignal                        string `hcl:"renew_signal"`
 	RenewSignalDeprecated              string `hcl:"renewSignal"`
+	FederatedTrustDomains              bool   `hcl:"federated_trust_domains"`
```

Comment:
I used `include_federated_domains` to keep the indentation in the Readme file, otherwise it wouldn't fit. Let me know if you think that `include_federated_trust_domains` is better.